RNSound.podspec

```
@@ -14,7 +14,7 @@ Pod::Spec.new do |s|
   s.requires_arc        = true
   s.platform            = :ios, "7.0"
   
-  s.dependency 'React/Core'
+  s.dependency 'React-Core'
```

How about doing the same as React Native Community libraries and use just `React` pod? https://github.com/react-native-community/react-native-webview/blob/master/react-native-webview.podspec#L18
```suggestion
  s.dependency 'React'
```

---

Ah this is good :) I was already wondering about the backwards compatibility of this change. But this should work fine.